usaspending_api/common/helpers.py

```diff
@@ -55,13 +55,20 @@ def generate_fiscal_month(date):
 def get_pagination(results, limit, page, benchmarks=False):
```

Comment:
Here we're bypassing the default django pagination we've previously utilized (`usaspending-api/common/pagination.py`). Is that primarily due to what the default pagination operates on? Default I believe operates on the queryset where as this operates on the serialized results (the JSON formatted resultset)?

Reply:
Yes, that's correct. `usaspending-api/common/pagination.py`'s code is hardcoded for querysets whereas this function is designed for any python list.